### PR TITLE
adding 'additional comment field to mental health section'

### PR DESCRIPTION
**Pull request by Dutervil**
adding 'additional comment field to mental health section
@lnball , Please have a look

### CODE REVIEW

**Comment by mogoodrich** [comment]
Thanks @Dutervil !  @lnball  in on vacation until next week.  This looks good to me, but I will defer to Ellen as she knows this code better.  That being said, if you need this merged in sooner than next week. let me know and I can approve and we can follow up with Ellen when she is back.

**Comment by Dutervil** [comment]
I think you can merge @mogoodrich , ask you say, Ellen can look at it when She is back

File: configuration/pih/htmlforms/mentalHealth.xml
Comment (mogoodrich) [inline configuration/pih/htmlforms/mentalHealth.xml:921]: this id seems inaccurate, as this isn't a radio button container, was this copied from somewhere else?

File: configuration/pih/htmlforms/mentalHealth.xml
Comment (mogoodrich) [inline configuration/pih/htmlforms/mentalHealth.xml:929]: unless we really need to keep it for some reason, let's just remove this instead of commenting it out?

**Review by mogoodrich — CHANGES_REQUESTED** [review]
Thanks @Dutervil ... I am happy to merge and have Ellen review when she gets back, but I just found two small things, can you take a look? Thanks!

File: configuration/pih/htmlforms/mentalHealth.xml
Comment (Dutervil) [reply configuration/pih/htmlforms/mentalHealth.xml:921]: I have created this id 'radio_button_container' because i wanted to fix the display of the field

File: configuration/pih/htmlforms/mentalHealth.xml
Comment (Dutervil) [reply configuration/pih/htmlforms/mentalHealth.xml:929]: what you ask me to remove is what we neek i think @mogoodrich

File: configuration/pih/htmlforms/mentalHealth.xml
Comment (mogoodrich) [reply configuration/pih/htmlforms/mentalHealth.xml:921]: Ah, okay, I see why I was confused.  Element ids should always be unque, but looks like in the mental health form there are many elements with the id "radio_button_container".  Could we change this to a class, ie


        .radio_button_container{
        display:flex;
        gap:50px;
        justify-content:space-between;
        }

... and then change all the id="radio_button_container" to class="radio_button container"?

File: configuration/pih/htmlforms/mentalHealth.xml
Comment (lnball) [inline configuration/pih/htmlforms/mentalHealth.xml:935]: @Dutervil You are already using this concept (CIEL:163044) on line 907 of this file.  You cannot use the same concept twice unless it's grouped in an obsgroup/convenience set (ConvSet).

**Review by lnball — CHANGES_REQUESTED** [review]
@mogoodrich made good suggestions about using class instead of id.  

@Dutervil You added a text area with the same concept that you used previously.  That's not allowed.

File: configuration/pih/htmlforms/mentalHealth.xml
Comment (Dutervil) [reply configuration/pih/htmlforms/mentalHealth.xml:935]: @lnball What is the right concept to use, please?

Comment (Dutervil) [reply configuration/pih/htmlforms/mentalHealth.xml:935]: @lnball I was not sure about the conpcet , i did it while you were in holiday, Please let me know the right ticket to use pleae

**Review by lnball — APPROVED** [review]
Looks good.  Thanks.

**Pull request merged**
